cogs/helpers/_taskValidator.py

```diff
@@ -45,19 +45,19 @@ def validate(fileName):
                 args = task[2]
                 preposition = task[3]
                 end = task[4].split()
-                out += f"Execute command `{command} {args}` at {start[1] if start[1] != '*' else 'any'}:{start[0] if start[0] != '*' else 'any'}"
+                out += f"Execute command `{command} {args}` at {start[1] if start[1] != '*' else 'any month'}:{start[0] if start[0] != '*' else 'any day'}"
                 out += f" on day {start[2] if start[2] != '*' else 'any'} of {months[start[3]]} if it is {days[start[4]]}day\n"
 
                 if preposition == "until":
                     out += f"    Revert command `{command} {args}` at {end[1] if end[1] != '*' else 'any'}:{end[0] if end[0] != '*' else 'any'}"
-                    out += f" on day {end[2] if end[2] != '*' else 'any'} of {months[end[3]]} if it is {days[end[4]]}day\n\n"
+                    out += f" on day {end[2] if end[2] != '*' else 'any day'} of {months[end[3]]} if it is {days[end[4]]}day\n\n"
 
                 # Add custom EOL character to split on in output command
                 out += chr(255)
 
             return (
                 True,
-                "File is valid. Tasks will run at the following times:\n\n" + out,
+                "File is valid. Tasks will run at the following times:\n'0'\n" + out,
```

Comment:
I am not sure what the purpose of these '0' 's are, could you explain what you were trying to do here? :)